gluon-connect-file-provider/pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.gluonhq.samples.connect</groupId>
```

Comment:
groupId should be just `com.gluonhq.samples`?

Reply:
For connect samples, we are appending samples with connect. I think it makes sense and gives samples an additional layer of division. Thoughts?

Reply:
I'll keep everything under samples. Some samples can apply to different layers.

Reply:
Or, we could put those samples under `mobile` ?

Reply:
Not that we are going to publish any of these, so I don't think it doesn't really matter. Imo, I'ld keep it simple.

Reply:
ok